website/StandardAttributes.rst

```diff
@@ -0,0 +1,1388 @@
+       </p>
+     </td>
+     <td style="vertical-align: top; width:100px"> required </td>
+   </tr>
+   
+   </table>
+   </embed>
+
+Multi-Part and Deep Data
+========================
+
+.. raw:: html
+
+   <embed>
+   <table>
+   
+   <tr>
+     <td style="vertical-align: top; width:150px; font-weight:bold"> attribute name</td>
+     <td style="vertical-align: top; width:100px; font-weight:bold"> type </td>
+     <td style="vertical-align: top; width:500px; font-weight:bold"> definition </td>
+     <td style="vertical-align: top; width:100px; font-weight:bold"> status </td>
+   </tr>
+   <tr style="border-width:3px">
+     <td style="vertical-align: top; width:150px"> <tt> <b> name </b> </tt> </td>
+     <td style="vertical-align: top; width:100px"> <tt> string </tt> </td>
+     <td style="vertical-align: top; width:500px">
+       <p style="padding-bottom:15px">
+         The <tt>name</tt> attribute defines the name of each part. The name of each
+         part must be unique. Names may contain <tt>'*.*'</tt> characters to present a
+         tree-like structure of the parts in a file.
+       </p>
+     </td>
+     <td style="vertical-align: top; width:100px"> required for multi-part images </td>
+   </tr>
+   <tr>
+     <td style="vertical-align: top; width:150px"> <tt> <b> type </b> </tt> </td>
+     <td style="vertical-align: top; width:100px"> <tt> string </tt> </td>
+     <td style="vertical-align: top; width:500px">
+       <p style="padding-bottom:15px">
+         Data types are defined by the type attribute. There are four
+         types:
+         <ul>
+           <li> Scan line images: indicated by a value of <tt>scanlineimage</tt> </li>
+           <li> Tiled images: indicated by a value of <tt>tiledimage</tt> </li>
+           <li> Deep scan line images:  indicated by a value of <tt>deepscanline</tt> </li>
+           <li> Deep tiled images:  indicated by a value of <tt>deeptile</tt> </li>      
+         </ul>
+       </p>
+     </td>
+     <td style="vertical-align: top; width:100px"> required for multi-part images </td>
+   </tr>
+   <tr>
+     <td style="vertical-align: top; width:150px"> <tt> <b> version </b> </tt> </td>
+     <td style="vertical-align: top; width:100px"> <tt> int </tt> </td>
+     <td style="vertical-align: top; width:500px">
+       <p style="padding-bottom:15px">
+         Version 1 data for all part types is described in the section on the OpenEXR File Layout.
+       </p>
+     </td>
+     <td style="vertical-align: top; width:100px"> required for multi-part images </td>
+   </tr>
+   <tr>
+     <td style="vertical-align: top; width:150px"> <tt> <b> chunkCount </b> </tt> </td>
+     <td style="vertical-align: top; width:100px"> <tt> int </tt> </td>
+     <td style="vertical-align: top; width:500px">
+       <p style="padding-bottom:15px">
+        Indicates the number of chunks in this part.  Required if the
+        multipart bit (12) is set.
```

Comment:
Also worth adding that it is created automatically by the library: the user does not need to compute it

Reply:
done.